components/google_drive/actions/add-file-sharing-preference/add-file-sharing-preference.js

```diff
@@ -0,0 +1,96 @@
+const googleDrive = require("../../google_drive.app");
+
+module.exports = {
+  key: "google_drive-add-file-sharing-preference",
+  name: "Add File Sharing Preference",
+  description: "Add File Sharing Preference",
```

Comment:
Is there a document you can link to here that describes what file sharing preferences are, or otherwise expands on what you have in `name`?

Reply:
Updated the [description](https://github.com/PipedreamHQ/pipedream/pull/1718/files#diff-285e469c2f1ee33054a35177fab40ba02a5bfb76cd9663ed31efd25c100be8beR12-R13) to be more descriptive.